### PR TITLE
Login

### PR DESCRIPTION
**Pull request by MadsDueKristensen**

### CODE REVIEW

File: src/domain/AuthManager.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+public class AuthManager {
+
+  /**
+   * Verify login
+   *
+   * @param username
+   * @param password
+   * @return true if ok
+   */
+  public static boolean vertify(String username, String password) {
```
Comment (kvartborg) [inline src/domain/AuthManager.java]: Should be `verify`.

File: src/domain/DomainFacade.java [hunk 2]
```diff
@@ -47,4 +47,9 @@ public void inject(IData data) {
     this.data = data;
   }
 
+  @Override
```
Comment (kvartborg) [inline src/domain/DomainFacade.java]: Docblock?

File: src/gui/controller/LoginController.java [hunk 3]
```diff
@@ -1,14 +1,40 @@
...
    * Reference to the label instance
    */
   @FXML
-  private Label label;
+  private TextField usernameField;
+  @FXML
+  private TextField passwordField;
+
+  @FXML
+  private void loginAction(ActionEvent event) {
```
Comment (kvartborg) [inline src/gui/controller/LoginController.java]: Docblock?

File: src/gui/controller/LoginController.java [hunk 4]
```diff
@@ -1,14 +1,40 @@
...
 
 public class LoginController {
 
   /**
    * Reference to the label instance
    */
   @FXML
-  private Label label;
+  private TextField usernameField;
+  @FXML
```
Comment (kvartborg) [inline src/gui/controller/LoginController.java]: Docblock?

File: src/acq/IGUI.java [hunk 5]
```diff
@@ -1,22 +1,18 @@
...
- * and open the template in the editor.
- */
 package acq;
 
-import javafx.stage.Stage;
-
 public interface IGUI {
+
   /**
    * Ibject domain
```
Comment (kvartborg) [inline src/acq/IGUI.java]: Should be corrected.

File: src/gui/GUI.java [hunk 6]
```diff
@@ -31,10 +24,7 @@
    */
   private IDomain domain;
 
-  /**
-   * Stage instance
-   */
-  private Stage stage;
+  private Map<String, Controller> controllers = new HashMap<>();
```
Comment (kvartborg) [inline src/gui/GUI.java]: Missing docblock

File: src/gui/controller/Controller.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+package gui.controller;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+import javafx.stage.Stage;
+
+/**
+ *
+ * @author yodamester42
```
Comment (grammeaway) [inline src/gui/controller/Controller.java]: Remove author-tag and license header

File: src/gui/controller/OverviewController.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+package gui.controller;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+/**
+ * FXML Controller class
+ *
+ * @author yodamester42
```
Comment (grammeaway) [inline src/gui/controller/OverviewController.java]: Remove author-tag and license header

**Pull request merged**